tools/walletextension/static/javascript.js

```diff
@@ -56,7 +56,7 @@ const initialize = () => {
             return
         }
 
-        const signedViewingKeyJson = {"signature": signature}
+        const signedViewingKeyJson = {"signature": signature, "address": account}
```

Comment:
Need to mention this to Moray again if you haven't already, in case he stopped sending it

Reply:
👍